cmd/flowlogs2metrics/main.go

```diff
@@ -89,8 +90,8 @@ func initLogger() {
 }
 
 func dumpConfig() {
-	configAsJSON, _ := json.MarshalIndent(config.Opt, "", "\t")
-	log.Infof("configuration:\n%s\n", configAsJSON)
+	configAsJSON, _ := json.MarshalIndent(config.Opt, "", "    ")
+	fmt.Printf("Using configuration:\n%s\n", configAsJSON)
```

Comment:
Moving this to be printf so we can always get this even if debug is in error (very useful for Palmetto)